packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -274,7 +274,9 @@ const VirtualListBaseFactory = (type) => {
 
 			if (containerNode && containerNode.removeEventListener) {
 				containerNode.removeEventListener('keydown', this.onKeyDown);
-				containerNode.removeEventListener('keyup', this.resumeSpotlight);
+			}
+			if (document) {
```

Comment:
Suggest we update this to use `this.resumeSpotlight()` as well so we don't accidentally leave spotlight paused when unmounting. Also gets DRYness bonus points. :)